semanticdb/scalac/library/src/main/scala/scala/meta/internal/semanticdb/scalac/DocumentOps.scala

```diff
@@ -189,7 +193,44 @@ trait DocumentOps { self: DatabaseOps =>
               todo -= mtree
 
               names(mtree.pos) = symbol
-              if (mtree.isDefinition) binders += mtree.pos
+              if (mtree.isDefinition) {
+                val isToplevel = {
```

Comment:
This logic does not make sense to me, would `isToplevel=true` for `x` in something like
```scala
object a {
  locally {
    class A {
      val x = 2 // toplevel=true ?
    }
  }
}
```

Would it be sufficient to check `gsym.owner.hasPackageFlag`? I don't recall the package index containing method symbols.